kratos/sources/connectivity_preserve_modeler.cpp

```diff
@@ -122,17 +102,15 @@ void ConnectivityPreserveModeler::CopyCommonData(
     rDestinationModelPart.AddNodes(rOriginModelPart.NodesBegin(), rOriginModelPart.NodesEnd());
 }
 
-
 void ConnectivityPreserveModeler::DuplicateElements(
-    ModelPart &rOriginModelPart,
-    ModelPart &rDestinationModelPart,
-    Element const &rReferenceElement)
+    ModelPart& rOriginModelPart,
```

Comment:
can't we make const the origin modelpart?

Reply:
I tried, but not possible bcs some methods are not const, e.g. [those](https://github.com/KratosMultiphysics/Kratos/blob/d5b63540b10a651f110906e0967a4c5707dbd454/kratos/sources/connectivity_preserve_modeler.cpp#L97-L99)